Connectors/Autocad/Speckle.Connectors.AutocadShared/Plugin/AppUtils.cs

```diff
@@ -10,21 +10,19 @@ public static class AppUtils
 #elif AUTOCAD
     HostApplications.AutoCAD;
 #else
-    throw new NotImplementedException();
+    throw new NotSupportedException();
```

Comment:
Just a comment,
I think we could use the preprocessor error 
```
#error host application not supported
```
instead, it might give us quicker feedback when we come to add a new project verison